iroha_2_whitepaper.md

```diff
@@ -79,14 +78,7 @@ Iroha uses a simple data model made up of domains, peers, accounts, assets, sign
 
 ### 2.6. Transactions
 
-Iroha supports the smart contract lifecycle transactions:
-
-* Chaincode deploy
-* Chaincode invoke
-* Chaincode update
-* Chaincode deprecate
-
-In addition to chaincode transactions, Iroha supports the following basic transaction types to support common asset management use cases:
+Iroha ships with a library of smart contracts, called the Iroha Special Instructions. These include the following basic transaction types to support asset management use cases:
```

Comment:
list below looks more like a list of commands than list of transaction types

Reply:
I agree, it's confusing to mess up transactions (which are just wrappers) and real commands/ISIs.